### PR TITLE
README: Add a warining of UNMAINTAINED

### PR DESCRIPTION
**Pull request by xescugc**
So it's easy to identify and where to go if a similar library it's neede

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Raws: AWS Reader [![Build Status](https://travis-ci.org/cycloidio/raws.svg?branch=master)](https://travis-ci.org/cycloidio/raws) [![Coverage Status](https://coveralls.io/repos/github/cycloidio/raws/badge.svg)](https://coveralls.io/github/cycloidio/raws)
 
+## UNMAINTAINED (18/12/2019)
+
+For several reasons we moved this lib inside of https://github.com/cycloidio/terracognita ([PR](https://github.com/cycloidio/terracognita/pull/71)), and we are making this as **UNMAINTAINED**, so if you want to use a similar one, it'll be https://github.com/cycloidio/terracognita/tree/master/aws/reader.
+
+It has several different things and it's development it's to fit TerraCognita needs, the main difference is that it does not support multi regions anymore, it was a blocking point if you wanted to use the AWS pagination to fetch more results (which we need on TerraCognita)
```
Comment (xlr-8) [inline README.md]: I'd rephrase:

```
It has several differences and its development is made to fit TerraCognita needs. The main difference being no more multi-region, as it was blocking for fetching more results due to pagination.
```

**Review by xlr-8 — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by xescugc** [comment]
Done

**Review by xlr-8 — APPROVED** [review]

**Comment by xlr-8** [comment]
RS & good to go
